docs/snippets/react/button-story-with-emojis.mdx.mdx

```diff
@@ -7,18 +7,17 @@ import { Button } from './Button';
 
 <Meta title="Components/Button" component={Button}/>
 
+<!-- The render function is a framework specific construct to tell how the story should render -->
```

Comment:
```suggestion
<!-- The render function is a framework specific construct to define how the story should render -->
```

Reply:
Seems like this comment should be present for the "JS" and "TS" versions of this snippet, too.

Reply:
Fixed with the latest commits. Thanks for spotting this!